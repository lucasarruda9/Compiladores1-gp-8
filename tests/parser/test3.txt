# Teste: atribuição e uso de variável
x = 42
y = x + 8
y * 999
