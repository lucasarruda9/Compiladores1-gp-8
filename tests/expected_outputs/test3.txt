Digite expressões em Python. Pressione Ctrl+D (Linux/Mac) ou Ctrl+Z seguido de Enter (Windows) para encerrar.
Atribuicao
    Variavel: x
    Literal: 42
Resultado da atribuição: 42
Atribuicao
    Variavel: y
    Operacao: +
        Variavel: x
        Literal: 8
Resultado da atribuição: 50
Operacao: *
    Variavel: y
    Literal: 2
Resultado: 100

=== Tabela de Símbolos ===
Nome: x, Tipo: int = 42
Nome: y, Tipo: int = 50
========================

